pkg/shell/superuser.jsx

```diff
@@ -108,7 +110,12 @@ const UnlockDialog = ({ proxy, host }) => {
 
     function init() {
         return proxy.Stop().always(() => {
-            start("sudo");
+            if (proxy.Bridges.length == 1)
+                start(proxy.Bridges[0]);
+            else {
+                setBridges(proxy.Bridges);
+                setBridge(proxy.Bridges[0]);
+            }
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-104722-284c800d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)